axelrod/strategies/memorytwo.py

```diff
@@ -1,11 +1,203 @@
+"""Memory Two strategies."""
+
+import itertools
+import warnings
+from typing import Tuple, Dict
+
 from axelrod.action import Action
 from axelrod.player import Player
+from axelrod.random_ import random_choice
 from .titfortat import TitForTat, TitFor2Tats
 from .defector import Defector
 
+
 C, D = Action.C, Action.D
 
 
+class MemoryTwoPlayer(Player):
+    """
+    Uses a sixteen-vector for strategies based on the last two rounds of play,
+    1.  P(C|CC, CC)
```

Comment:
Maybe better as (or something similar to): 
```Based on the 16 conditional probabilities P(X | I,J,K,L) where X, I, J, K, L in [C, D] and I, J are the players last two moves and K, L are the opponents last two moves.```

Reply:
I think I'd prefer them explicitly listed, the ordered isn't clear otherwise?

Reply:
Unless your suggestion is to use your sentence to clarify what the list is and also list them which is a great call :+1: